docs/reference_manual/appendix/dashboard_how_to_guide.rst

```diff
@@ -128,16 +128,16 @@ Manifest Tab
 ============
 
 The next tab you can select is the manifest tab. This displays the manifest after it has been filtered through to make it more readable.
-More specifically, if the 'pernode' value of the leaf of the Schema is 'never', the value of the leaf
+More specifically, if the pernode value of the leaf of the Schema is :term:`pernode='never'`, the value of the leaf
 is the value of the leaf['node']['global']['global']['value']. If there is no value for that, then 
 it is the value of the leaf['node']['default']['default']['value']. Outside of that,
-the nodes will be concatenated, or if the step and index is 'default' and 'default' or 'global' and 'global',
-the node will be 'default' or 'global', respectively. 
+the nodes will be concatenated, or if the step and index is :term:`step='default'` and :term:`index='default'`
+or :term:`step='global'` and :term:`index='global'`, the node will be :term:`node='default'` or :term:`node='global'`, respectively. 
```

Comment:
I think this should just be :term:`pernode` and :term:`global`, these don't generate anything so I assume there are some warnings about it